sdk/storage/azure-storage-blob/azure/storage/blob/_shared/uploads.py

```diff
@@ -56,24 +50,9 @@ def upload_data_chunks(
         max_concurrency=None,
         stream=None,
         validate_content=None,
-        encryption_options=None,
         progress_hook=None,
         **kwargs):
 
-    if encryption_options:
```

Comment:
This code needed to be moved to the Blob specific code (in `upload_helpers` since encryption was no longer in the shared folder. It was also good to get this encryption stuff out of shared as well.